src/main/kotlin/blackjack/domain/card/Cards.kt

```diff
@@ -0,0 +1,37 @@
+package blackjack.domain.card
+
+data class Cards(
+    private val _cards: MutableList<Card> = mutableListOf(),
+) {
+    val cards: List<Card>
+        get() = _cards
```

Comment:
```suggestion
        get() = _cards.toList()
```
위와 같이 리스트 객체의 원본이 바뀌더라도 새로운 값을 참조하도록 방어적 복사를 적용해보면 어떨까요?

다음 글이 도움되면 좋겠어요!
[Kotlin의 방어적 복사, 뼛속까지 안전하게 복사하기](https://velog.io/@hyemdooly/Kotlin%EC%9D%98-%EB%B0%A9%EC%96%B4%EC%A0%81-%EB%B3%B5%EC%82%AC-%EB%BC%9B%EC%86%8D%EA%B9%8C%EC%A7%80-%EC%95%88%EC%A0%84%ED%95%98%EA%B2%8C-%EB%B3%B5%EC%82%AC%ED%95%98%EA%B8%B0)